<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="libraryTab">
       <attribute name="title">
        <string>Library</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>160</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="peopleLabel">
           <property name="text">
            <string>Currently reading: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="peopleCountLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="booksTab">
       <attribute name="title">
        <string>Books</string>
       </attribute>
       <widget class="QPushButton" name="sortButton">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>120</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string> Sort by</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addBookButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>add Book</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeBookButton">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>280</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>remove Book</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="sortByComboBox">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>120</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>book_name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>author</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>publisher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>publishing_date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>book_group</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>category</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="propertiesButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>280</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>properties</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="searchByComboBox">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>0</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>book_name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>author</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>publisher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>book_group</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>category</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>0</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Search by</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>330</y>
          <width>601</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>author</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>group</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>category</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>publisher</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>publish date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>state</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>id_hidden</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
